test_syndiffix/testUtils.py

```diff
@@ -61,7 +61,9 @@ def makeMlClassInfo(self, df, dataSource, columns=None):
     def getColTypes(self, df):
         colTypes = []
         for colType in df.dtypes:
-            if pd.api.types.is_integer_dtype(colType):
+            if pd.api.types.is_bool_dtype(colType):
+                colTypes.append('bool')
```

Comment:
here you append 'bool', but in line 139 you append 'boolean'. Better to make both of the 'boolean'?

Reply:
ah, right, I adhered to two different conventions here (pandas vs SynDiffix), but it doesn't make a difference in the logic. Fixing...